frontend/src/components/Chat.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import { withTranslation, Trans} from "react-i18next";
 import {Button, Link, Badge, TextField, Grid, Container, Card, CardHeader, Paper, Avatar, FormHelperText, Typography} from "@mui/material";
 import ReconnectingWebSocket from 'reconnecting-websocket';
+import * as openpgp from 'openpgp/lightweight';
```

Comment:
I see this being imported, but it never gets used. Is it supposed to be like this?

Reply:
Sorry, this is work in progress. But somehow that commit also slipped trough bad "git checkout -b newbranch" into another feature branch that is already merged into master :)